feature-libs/product-configurator/rulebased/components/conflict-solver-dialog/configurator-conflict-solver-dialog.component.html

```diff
@@ -0,0 +1,176 @@
+            [currentGroup]="conflictGroups[0]"
+          ></cx-configurator-conflict-description>
+          <div
+            class="cx-group-attribute"
+            *ngFor="
+              let attribute of conflictGroups[0].attributes;
+              let indexOfAttribute = index
+            "
+          >
+            <ng-container *ngIf="activeLanguage$ | async as activeLanguage">
+              <cx-configurator-conflict-suggestion
+                [currentGroup]="conflictGroups[0]"
+                [attribute]="attribute"
+                [suggestionNumber]="indexOfAttribute"
+              ></cx-configurator-conflict-suggestion>
+              <cx-configurator-attribute-header
+                [attribute]="attribute"
+                [owner]="routerData.owner"
+                [groupId]="conflictGroups[0].id"
+                [groupType]="conflictGroups[0].groupType"
+                [isNavigationToGroupEnabled]="false"
+              >
+              </cx-configurator-attribute-header>
+              <!-- Single Selection Radio Button -->
+              <cx-configurator-attribute-radio-button
+                *ngIf="
+                  attribute.uiType === uiType.RADIOBUTTON ||
+                  attribute.uiType === uiType.RADIOBUTTON_ADDITIONAL_INPUT
+                "
+                [attribute]="attribute"
+                [ownerKey]="routerData.owner.key"
+                [ownerType]="routerData.owner.type"
+                [language]="activeLanguage"
+                (selectionChange)="updateConfiguration($event)"
+              ></cx-configurator-attribute-radio-button>
+
+              <!-- Single Selection Product Bundle -->
+              <cx-configurator-attribute-single-selection-bundle
+                (selectionChange)="updateConfiguration($event)"
+                [attribute]="attribute"
+                [ownerKey]="routerData.owner.key"
+                *ngIf="attribute.uiType === uiType.RADIOBUTTON_PRODUCT"
+              ></cx-configurator-attribute-single-selection-bundle>
+
+              <!-- Multi Selection Product Bundle -->
+              <cx-configurator-attribute-multi-selection-bundle
+                (selectionChange)="updateConfiguration($event)"
+                [attribute]="attribute"
+                [ownerKey]="routerData.owner.key"
+                *ngIf="attribute.uiType === uiType.CHECKBOXLIST_PRODUCT"
+              ></cx-configurator-attribute-multi-selection-bundle>
+
+              <!-- Single Selection Dropdown Product Bundle -->
+              <cx-configurator-attribute-single-selection-bundle-dropdown
+                *ngIf="attribute.uiType === uiType.DROPDOWN_PRODUCT"
+                [attribute]="attribute"
+                [group]="conflictGroups[0].id"
+                [ownerKey]="routerData.owner.key"
+                (selectionChange)="updateConfiguration($event)"
+              ></cx-configurator-attribute-single-selection-bundle-dropdown>
```

Comment:
i believe we can remove those, because bundles are only supported by CPQ and not by AVC.

Reply:
Agree!

Reply:
Done!